### ISSUE TITLE
Start value of the minimum

### ISSUE THREAD

**Issue by tinrabuzin** [beutlich/LastLib#1]
Hi, 

Thanks for the great library! I noticed after the latest commit that the computation of the minimum does not start from the specified `y_start` value. See the screenshot bellow. I get this result on Linux OM v1.16.5. 

![image](https://user-images.githubusercontent.com/13916934/110382721-97acde00-805b-11eb-8f08-a6ff04b02fcd.png)

**Comment by tinrabuzin** [comment]
@beutlich I don't know if this is the _good_ fix, but I was playing around with the models and realized that if I write in the line 263 of `LastLib.Last_omc` the following:

```modelica
  flowParamsStart := fmi1Functions.fmi1SetRealParameter(fmi1me, {1.0}, {y_start});
```
instead of 
```modelica
  flowParamsStart := fmi1Functions.fmi1SetRealParameter(fmi1me, {0.0}, {y_start});
```
the example starts from the correct `y_start`.

**Comment by beutlich** [comment]
Sorry, I missed this one completely.

Can you share the model `MinimumTest.Test` please that you edited with OMEdit? Maybe the workaround mentioned on page 13 of https://tinyurl.com/lastlibfmu still needs to be applied.

**Comment by tinrabuzin** [comment]
I haven't changed `MinimumTest.Test`. It was `LastLib.Last_omc` that I was changing. I'll make a draft PR if you want?

**Comment by beutlich** [comment]
Still you need to adapt MinimumTest.Test to select the redeclared model of Last being LastLib.Last_omc.

**Comment by tinrabuzin** [comment]
Right, I think I did that already:
```modelica
block Minimum
  extends Modelica.Blocks.Interfaces.SISO;
  parameter Real y_start=Modelica.Constants.inf "Initialization of minimum";
  replaceable model Last = LastLib.Last_omc constrainedby LastLib.LastBase(y_start=y_start) "Last FMU"
    annotation(choices(
     choice(redeclare model Last = LastLib.Last_dymola_linux "Last for Dymola (Linux)"),
     choice(redeclare model Last = LastLib.Last_dymola_windows "Last for Dymola (Windows)"),
     choice(redeclare model Last = LastLib.Last_omc "Last for OpenModelica (C runtime)"),
     choice(redeclare model Last = LastLib.Last_simx "Last for SimulationX")));
  Last last(y_start=y_start) "Last FMU" annotation(Placement(transformation(extent={{-30,10},{-10,30}})));
  Modelica.Blocks.Math.Min min annotation(Placement(transformation(extent={{10,-10},{30,10}})));
  equation
    connect(u, min.u2) annotation(Line(points={{-120,0},{0,0},{0,-6},{8,-6}}, color={0,0,127}));
    connect(min.y, y) annotation(Line(points={{31,0},{110,0}}, color={0,0,127}));
    connect(last.y, min.u1) annotation(Line(points={{-9,20},{0,20},{0,6},{8,6}}, color={0,0,127}));
    connect(min.y, last.u) annotation(Line(points={{31,0},{40,0},{40,40},{-40,40},{-40,20},{-32,20}}, color={0,0,127}));
  annotation(defaultComponentName="min", preferredView="diagram");
end Minimum;
```

**Comment by beutlich** [comment]
OK, I have another idea:

```diff
--- a/LastLib/Resources/sources/last.c
+++ b/LastLib/Resources/sources/last.c
@@ -34,7 +34,7 @@ void setStartValues(ModelInstance *comp) {
 // called by fmiInitialize() after setting eventInfo to defaults
 // Used to set the first time event, if any.
 void initialize(ModelInstance* comp, fmiEventInfo* eventInfo) {
-    r(y_) = r(y_start_);
+    r(u_) = r(y_start_);
 }
 
 // called by fmiCompletedIntegratorStep()
```

**Comment by beutlich** [comment]
I can provide the updated FMU binaries if you want to test it,

**Comment by tinrabuzin** [comment]
Yes, please do!

**Comment by beutlich** [comment]
They are now available from new branch [fix-y_start](https://github.com/beutlich/LastLib/tree/fix-y_start).

**Comment by beutlich** [comment]
But fail to correctly initialize in Dymola ...

**Comment by tinrabuzin** [comment]
My Linux machine is temporarily not available, but OMC on Windows also does not initialize correctly with the new binaries.

**Comment by beutlich** [comment]
... and pushed again.

**Comment by beutlich** [comment]
OK, need to check in more detail later,

**Comment by tinrabuzin** [comment]
I deleted my comment since I was doing things in a wrong terminal. I checkout out the latest commit in the branch and it works now! At least on Windows OMC. Thanks!

**Comment by beutlich** [comment]
Phew, thanks for confirmation. If you can test in Linux later, just let me know, if it also works there,

**Comment by tinrabuzin** [comment]
It works on Linux as well!

**Comment by beutlich** [comment]
Resolved by 65f205f.
